src/instance.ts

```diff
@@ -140,6 +141,12 @@ export type SetInstanceMetadataCallback = (
 ) => void;
 export type SetInstanceMetadataResponse = [google.protobuf.Empty];
 
+export interface CreateTableFromBackupConfig {
+  table: string;
```

Comment:
Should we split `tableId` param from the rest of the config values to have similar feel as `Instance.createTable()`?

Reply:
That is actually how it used to be, and I changed it to the config object. I didn't think about matching "createTable(tableId)", although that makes sense. However, I still think it's best to do a configuration object, as my reasoning was it's not intuitive from the method name which argument should come first, "table" or "backup". Each would be logical and could trip users up:

```js
await createTableFromBackup('my-backup', 'my-table') // :(
await createTableFromBackup('my-table', 'my-backup') // :(
await createTableFromBackup({table: 'my-table', backup: 'my-backup'}) // :)
```

LMK if you still think we should switch!

Reply:
Let's have the config object 👍